x-pack/test/api_integration/apis/monitoring/setup/collection/index.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default function ({ loadTestFile }) {
+  describe('Collection', () => {
+    loadTestFile(require.resolve('./kibana_mb'));
+    loadTestFile(require.resolve('./kibana_exclusive_mb'));
+    loadTestFile(require.resolve('./es_and_kibana_mb'));
+    loadTestFile(require.resolve('./es_and_kibana_exclusive_mb'));
+    loadTestFile(require.resolve('./detect_beats'));
```

Comment:
You've got ES, Kibana, and Beats covered here. Do you think we should add coverage for APM and Logstash too?

Reply:
Yup definitely should. Thanks for the reminder

Reply:
To be clear, this would just cover the detection logic. I'm not sure we can test between `-mb-` indices and non `-mb-` indices for beats, logstash or APM? Am I understanding that correctly? Because beats and APM won't be generating those indices and logstash doesn't support that yet

Reply:
Your understanding is accurate.

At some point (hopefully soon) we'll have `-mb-` indices for Logstash. At that time we'd want to come back and add coverage for those here. For a future PR, of course!